### ISSUE TITLE
Documentation: On MacOS the terminal will not accept the full header paste

### ISSUE THREAD

**Issue by phirestalker** [sigma67/ytmusicapi#150]
MacOS terminal does not accept paste data above 1024 characters (I believe that is the number). The user will need to make use of pbpaste to get the headers "pasted" into the terminal.

I propose adding a note to the documentation for MacOS users that informs them to prepend their script command with `pbpaste |`

**Comment by sigma67** [comment]
Perhaps it makes sense to put this in the info printout, based on the detected platform?

https://github.com/sigma67/ytmusicapi/blob/4f4818f9263c8b58779e8564c95d7f22015b88c0/ytmusicapi/setup.py#L13

**Comment by phirestalker** [comment]
That would work. Then they could exit and rerun the script with pbpaste.

**Comment by sigma67** [comment]
Ah nevermind then, it should be in the docs as you said. We can add a little dropdown box "Mac OS" after the first code fragment here: https://ytmusicapi.readthedocs.io/en/latest/setup.html#using-the-headers-in-your-project

Feel free to submit a PR, I don't have a Mac and don't know how this works.

**Comment by phirestalker** [comment]
Ya, I was just going to do that, but I don't know the format of the file. I thought it was just markdown, but it is different or extended. I'll look it up and see if I can figure out the format.

**Comment by sigma67** [comment]
It's ReStructured Text (RST), it's fairly straightforward

The dropdowns are made using a little HTML trick, you can just copy this fragment and modify it
```rst
.. raw:: html

   <details>
   <summary><a>Chromium (Chrome/Edge)</a></summary>

.. container::

    - Verify that the request looks like this: **Status** 200, **Type** xhr, **Name** ``browse?...``
    - Click on the Name of any matching request. In the "Headers" tab, scroll to the section "Request headers" and copy everything starting from "accept: \*/\*" to the end of the section

.. raw:: html

   </details><br>
```

**Comment by phirestalker** [comment]
Thanks, I got it submitted. I hope it is clear enough.

**Comment by sigma67** [comment]
Merged in a2f6617
